.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,34 @@
+name: Test build on Windows
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/build-on-windows.yml'
+      - RNReanimated.podspec
+      - android/build.gradle
+      - package.json
+  push:
+    branches:
+      - main
+
+env:
+  REACT_NATIVE_VERSION: "0.70.5"
```

Comment:
What about using `latest` instead? This should work automatically for patch versions and we will know earlier if next minor version breaks something.